tests/ens/test_wildcard_resolution.py

```diff
@@ -10,9 +10,37 @@ def test_wildcard_resolution_with_extended_resolver_for_subdomains(ens, subdomai
     assert resolved_child_address == "0x000000000000000000000000000000000000dEaD"
 
 
-def test_wildcard_resolution_with_extended_resolver_for_parent_ens_domain(ens):
+def test_async_wildcard_resolution_with_extended_resolver_for_parent_ens_domain(ens):
```

Comment:
This should be the regular version and `async` added to the test name on line 39, yes?

Reply:
💯 ty 👍